src/key.rs

```diff
@@ -115,7 +121,9 @@ impl str::FromStr for SecretKey {
     }
 }
 
-/// A Secp256k1 public key, used for verification of signatures.
+/// Public key - a point on the secp256k1 curve.
+///
+/// Obtained by multiplying a secret key scalar by G (the generator point).
```

Comment:
This is true, but I think it's a little too technical for a doccomment.

Reply:
No worries, removed.